src/layout/Pickers.js

```diff
@@ -52,7 +52,31 @@ export let ModalPicker = withStateLens({ isOpen: false })(
             }}
           />
         </Modal>
-        <Button onClick={F.on(isOpen)}>{label}</Button>
+        <Button onClick={F.on(isOpen)}>
+          {// React warns when the label is a component with children that don't
+          // have unique keys.  This is how we can force it. It's unlikely that
+          // we could be able to solve it differently, since the definition of
+          // uniqueness needs to happen at the rendering of the parent, and the
+          // label is rendered as a given property on the instatiation of our
+          // pickers (or their parent components).
+          typeof _.get('$$typeof', label) === 'symbol'
```

Comment:
Can we use `_.isSymbol` here?

Reply:
Also, if label is a symbol we treat it like an object? I'm confused what this is doing.

Reply:
This code is very cryptic - and if the issue can be solved by just passing unique keys from the calling location, that makes this a user error and we should just add keys there. The relevant part is probably here: https://github.com/smartprocure/contexture-react/blob/master/src/themes/greyVest.js#L554-L557 However, we don't see this error on the contexture-imdb grey vest story, so it could be somewhere else.

Reply:
I tried to comment why this code seemed like a good idea, but here's another take:
The `label` used by the Picker can be either a string or anything that could be passed to React. The way we're generating these components is very functional, so _I strongly believe_ React gets confused and thinks that the children of the `label` must have unique IDs (which doesn't always happen, specially doesn't happen when children are just written as HTML elements one below the other. It's common to see it when iterating an array).

Now, the alternative fix (which I pushed to this same branch a moment ago) is to just add these keys into `greyVest`, where it's needed. However: **it was considerably hard to track**, end users won't understand the problem, this shouldn't normally happen if you're adding elements within a `span` (while not iterating an array, for example). This is why this _symbol_ code seemed useful to me, it automatically fixes the unexpected necessity of a unique `key` property where it's not normally needed.